### PR TITLE
Creating ROCm whl upon release

### PR DESCRIPTION
**Pull request by gshtras**
Creating ROCm versions of vllm and gradlib whl

### CODE REVIEW

**Review by Alexei-V-Ivanov-AMD — APPROVED** [review]
.github/workflows/scripts/build.sh:L4

Definitely "python_executable=python$1" is a cryptic way to interface the desirable python flavor (not ideal), but why we limit the script to python3 ?

Otherwise it looks good to me.

File: .github/workflows/scripts/build.sh [hunk 1]
```diff
@@ -1,23 +1,22 @@
 #!/bin/bash
 set -eux
 
-python_executable=python$1
```
Comment (Alexei-V-Ivanov-AMD) [inline .github/workflows/scripts/build.sh:4]: Definitely "python_executable=python$1" is a cryptic way to interface the desirable python flavor (not ideal), but why we limit the script to python3 ?

**Review by Alexei-V-Ivanov-AMD — DISMISSED** [review]
Besides the comment **it looks good to me.**

File: .github/workflows/scripts/build.sh (hunk 1 above)
Comment (gshtras) [reply .github/workflows/scripts/build.sh:4]: What other options are there in our ROCm release docker (or anywhere)?

File: .github/workflows/scripts/build.sh [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
+rm -f $(which sccache)
 
-bash tools/check_repo.sh
+export MAX_JOBS=32
 
 # Build
 $python_executable setup.py bdist_wheel --dist-dir=dist
+cd gradlib
+$python_executable setup.py bdist_wheel --dist-dir=dist
+cd ..
```
Comment (Alexei-V-Ivanov-AMD) [inline .github/workflows/scripts/build.sh:23]: This makes it perfect, doesn't it?

**Review by Alexei-V-Ivanov-AMD — DISMISSED** [review]

File: .github/workflows/scripts/build.sh (hunk 2 above)
Comment (gshtras) [reply .github/workflows/scripts/build.sh:23]: We'll see if whls will get published this time.

Comment (Alexei-V-Ivanov-AMD) [reply .github/workflows/scripts/build.sh:23]: That was the last time I approved without seeing functionality proof.

File: .github/workflows/scripts/build.sh (hunk 1 above)
Comment (Alexei-V-Ivanov-AMD) [reply .github/workflows/scripts/build.sh:4]: python2 ?

Comment (gshtras) [reply .github/workflows/scripts/build.sh:4]: This code will not be within the top 1000 things that will break :)

**Review by Alexei-V-Ivanov-AMD — APPROVED** [review]
Looks good to me.

**Pull request merged**